### PR TITLE
Feature/kirkstone community platforms

### PR DESCRIPTION
**Pull request by leenaS-d**

### CODE REVIEW

File: recipes-workaround/firebolt-native-sdk/firebolt-native-sdk.bb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Remove when RDK-53836 is resolved
+
+SUMMARY = "Native SDK"
+DESCRIPTION = "Recipe for building Native SDK"
+
+LICENSE = "CLOSED"
```
Comment (mhughesacn) [inline recipes-workaround/firebolt-native-sdk/firebolt-native-sdk.bb:6]: Should  not have an RDK-licensed component referenced in an OSS metalayer.

File: recipes-workaround/mountutils/mountutils.bb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+#To do - Remove when RDK-53835 is resolved
+#
+# ============================================================================
+# COMCAST C O N F I D E N T I A L AND PROPRIETARY
```
Comment (mhughesacn) [inline recipes-workaround/mountutils/mountutils.bb:4]: Recipes do not have a header, especially a confidential one.

File: recipes-workaround/rdk-gstreamer-utils-platform/rdk-gstreamer-utils-rpi_git.bb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+SUMMARY = "No Operation Recipe for secapi adapter version 2.0"
+LICENSE = "CLOSED"
```
Comment (mhughesacn) [inline recipes-workaround/rdk-gstreamer-utils-platform/rdk-gstreamer-utils-rpi_git.bb:2]: CLOSED license in an opensourced repo?

File: recipes-workaround/storagemanager/storagemanager_git.bb [hunk 4]
```diff
@@ -0,0 +1,89 @@
+# ============================================================================
+# RDK MANAGEMENT, LLC CONFIDENTIAL AND PROPRIETARY
```
Comment (mhughesacn) [inline recipes-workaround/storagemanager/storagemanager_git.bb:2]: - Confidential header in opensourced repo?
- Licensed component in opensourced repo?  Thisnrecipe doesn't belong here.

**Review by mhughesacn — CHANGES_REQUESTED** [review]
See comments.  Also, if enabling fdk-aac as a commercial component, you need to be certain that if images are distributed, recipients are sure of their license obligations.  (e.g. https://developer.garmin.com/open-source/fraunhofer/).

**Comment by mhughesacn** [comment]
Hi LeenaS, Arun: As Blackduck is now passing and all the awkward findings are gone, I don't have any comments here.  Thanks for making the changes.

**Comment by leenaS-d** [comment]
abandon

**Pull request closed without merging**